src/main/java/roomescape/controller/ReservationController.java

```diff
@@ -30,16 +30,16 @@ public String reservation() {
 
     @ResponseBody
     @GetMapping("/reservations")
-    public List<Reservation> reservations() {
+    public List<ReservationDto> reservations() {
         return reservationService.getAllReservations();
     }
 
     @ResponseBody
     @PostMapping("/reservations")
-    public ResponseEntity<Reservation> addReservation(@RequestBody Reservation reservation) throws URISyntaxException {
-        Reservation responseReservation = reservationService.addReservation(reservation);
-        return ResponseEntity.created(new URI("/reservations/" + responseReservation.id()))
-            .body(responseReservation);
+    public ResponseEntity<ReservationDto> addReservation(@RequestBody ReservationDto reservationDto) throws URISyntaxException {
+        ReservationDto responseReservationDto = reservationService.addReservation(reservationDto);
```

Comment:
위 내용과 마찬가지로 사소한 부분입니다만
변수명을 response정도로만 표현해도 괜찮지 않을까 싶어요

```suggestion
        ReservationDto response = reservationService.addReservation(request);
```